core/cap-0023.md

```diff
@@ -344,6 +341,20 @@ The behavior of `CreateClaimableBalanceOp` is as follows:
     - `asset` as specified in the operation
     - `amount` as specified in the operation
     - `reserve` equal to `claimants.size() * baseReserve`
+
+[^id-arithmatic]: ```
+    OperationID = (
+      sourceAccount ||
```

Comment:
I'm not familiar with this notation. It looks like OperationID can be any one of the values specified below.

Reply:
Conformed to struct style, albeit fairly repetitive of [primary definition](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0023.md?plain=1#L224-L230) which is challenging to rectify without introducing the `HashIDPreimage` explanations with the full context of this note.